tests/integration/techRecords.intTest.ts

```diff
@@ -314,25 +313,6 @@ describe("techRecords", () => {
               expect(res.body).toEqual("Technical Record created");
             });
           });
-
-          context("and that vehicle does exist", () => {
-            it("should return status 400 The conditional request failed", async () => {
```

Comment:
Why did you have to remove this test?

Reply:
Because now that systemNumber is automatically generated and you cannot predict what value it will have, I cannot create an integration test case where it will generate an existing systemNumber